design/src/main/kotlin/uk/govuk/app/design/ui/component/Header.kt

```diff
@@ -40,27 +42,94 @@ fun TabHeader(
     }
 }
 
+data class ActionButton(
+    val text: String? = null,
+    val onClick: () -> Unit
+)
+
 @Composable
 fun ChildPageHeader(
-    text: String,
-    onBack: () -> Unit,
+    text: String? = null,
+    backButton: ActionButton? = null,
+    actionButton: ActionButton? = null,
```

Comment:
I'm not sure what this is giving us for the back button seeing as the text is not being used. Could we not just pass an onBack param and display the back button based on the presence of it?

Reply:
Happy to replace it if you know how to test for an empty lambda. I tried and searched, but from what I can see it looks like you can't 🤷  I came up with a data class because at least you can test for nullable on them.

I could create a BackAction data class - it'll only have the onClick of course?

Reply:
You can make the lamda nullable:
onBack: (() -> Unit)?

Reply:
Refactored to use nullable lambda's (both onBack and onAction) - thanks @davidc-gds  👍